frontend/src/pages/LoginRegister/WelcomeGem.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <van-image :src="getGemImageUrl" width="32vw" height="32vw" />
+</template>
+
+<script lang="ts">
+import { defineComponent, PropType } from "vue";
+import { getEnumKeyByEnumValue } from "@/utils/enum";
+import { GemColor } from "@/interfaces";
+
+export default defineComponent({
+  props: {
+    gemColor: {
+      type: Object as PropType<GemColor>,
```

Comment:
```suggestion
import { defineComponent } from "vue";
import { getEnumKeyByEnumValue } from "@/utils/enum";
import { GemColor } from "@/interfaces";

export default defineComponent({
  props: {
    gemColor: {
      type: String,
```

To resolve warning in console:
```
[Vue warn]: Invalid prop: type check failed for prop "gemColor". Expected Object, got String with value "#99CEE0".
```